webapp/package.json

```diff
@@ -13,9 +13,13 @@
     "test": "jest"
   },
   "dependencies": {
-    "@emotion/react": "^11.11.1",
-    "@emotion/styled": "^11.11.0",
-    "@mui/joy": "^5.0.0-beta.11",
+    "@emotion/cache": "^11.11.0",
```

Comment:
Why do we add a direct dependency on `@emotion/cache`? I don't see any direct use of it.

Reply:
Removed in https://github.com/zetkin/lyra/pull/107/commits/1ec625a8204ffd2e0f29249d6ea54d0c9719700b,